framework/src/media/MediaPlayerImpl.cpp

```diff
@@ -605,8 +608,8 @@ void MediaPlayerImpl::playback()
 			}
 		}
 	} else if (num_read == 0) {
-		notifyObserver(PLAYER_OBSERVER_COMMAND_FINISHIED);
```

Comment:
`notifyObserver(PLAYER_OBSERVER_COMMAND_FINISHIED);` doesn't exist anywhere. Is it Okay?